Mod4JobFinder/FrontEnd
# Testing/error handling

# PR JobFinder
**Aaron F., Andrew J., Connor A.-L., Joseph B., Khoa N., Trevor S.**

### What does this do?


### Are there any known bugs?


### Summary of the changes
Add sad path tests for error handling in user login, new user form, and job search. Also add tests for navigation buttons on header.


### What ticket(s) do the changes resolve?
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
Most tests were added in the Sad_Path Cypress file but some changes were made to the other test file.


### How to Test Changes?
After pulling down and starting local host, run `npm run cypress` to open cypress test suite, check that all tests in both files are passing.


### Next Iterations
Improve Cypress testing for happy path/user flows.
